docs/chapter_phil/What is probability? Bayes contra Hume.rst

```diff
@@ -1,3 +1,801 @@
+Before we get a more detailed look at some important concepts, let us make a 
+short excursion into the philosophy of science and give these new concepts some 
+time to settle in your mind. Our excursion should make it clearer why so 
+many scientists had a hard time with subjective probability, even when most of 
+us are using it intuitively.
+
+
+A philosophical excursion to Popper & Hume
+==========================================
+
+When you think about the logic of the Bayesian approach, it is pretty close to 
+inductive thinking – the process to come up with rules from observations. Let 
+us take the famous swan argument here as an example. You see one white 
+swan; and another one; and another one; and so on, and you come to the 
+inductive conclusion that all swans are white. You have no guarantee that this 
+rule is true but due to your observations it seems plausible to you. You can do 
+the same thought experiment with the thought that you will wake up the next 
+morning or that the sun will rise. The school of thought that used inductive 
+thinking was called positivism and this thinking had two famous opponents – 
+David Hume (1711-1776) and Sir Karl Popper (1902-1994).
+
+David Hume was a Scottish philosopher who argued that we should never reason 
+from experience (seeing a lot of white swans) about situations we have not 
+experienced yet (seeing a swan of a different color). You might say that in 
+your experience the probability increases when you see tons of white swans that 
+the next one will be white too. Hume would disagree with that because it does 
+not follow logically. Take the second thought experiment – you waking up 
+in the morning. Every day you wake up in the morning and this experience should 
+increase the probability of you waking up tomorrow – inductively speaking. 
+Now, add age to the equation and you see at one point, it becomes less 
+likely that you will wake up the next morning. Hume points out that ‘no 
+matter how often induction has worked in the past, there is no reason to think 
+it will work ever again. Not unless you already assume induction, that is’ 
+(`Dienes, 2008`_, p.5). A historical fun fact that is closely related to the 
+swan argument, comes from the time that the British went to Australia. Guess 
+what they found? Of course, they found black swans.
+
+The second interesting mind here is Karl Popper, who wrote a lot about the 
+philosophy of science and what distinguishes science from non-science. Popper 
+argued against positivism and with it inductive thinking and his philosophy 
+was fallibilism. In a nutshell (because `Chapter 1.2`_ is dealing with this in 
+a much deeper way): You cannot say something is true, you can only falsify 
+statements. Popper agreed with Hume’s statements (e.g. `Popper, 1934`_). For 
+him a theory would in a best case scenario always be a guess, nothing more. 
+Maybe his reasoning had something to do with the fact that during his youth one 
```

Comment:
I don't like this "maybe". Popper says quite clearly in his biography 1. why he came to believe what he believed, 2. what the role of, e.g., the Eddington expedition was.

I would like this rephrased.

Reply:
It is also a bit weird to discuss Popper *after* null hypothesis rejection testing. Popper was not a Bayesian; but NHST is often justified in Popperian terms.